include/tvm/target/target.h

```diff
@@ -93,6 +94,13 @@ class TargetNode : public Object {
  private:
   /*! \brief Internal string repr. */
   mutable std::string str_repr_;
+  /*! \brief Parsing TargetNode::attrs from a list of raw strings. */
+  ObjectRef ParseAttr(const ObjectRef& obj, const TargetKindNode::ValueTypeInfo& info) const;
```

Comment:
document all arguments

Reply:
Will check all documents after the impl is done :-)